server/handler/base.go

```diff
@@ -193,5 +206,58 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, cl
 		return errors.Errorf("evaluation resulted in unexpected state: %s", result.Status)
 	}
 
+	if statusState == "pending" && prctx.IsDraft() != nil && !*prctx.IsDraft() {
+		subsetCurrentReviewers, _, err := client.PullRequests.ListReviewers(ctx, prctx.RepositoryOwner(), prctx.RepositoryName(), prctx.Number(), &github.ListOptions{
```

Comment:
Per above, for testing, I'd like to keep all read operations like this on `pull.Context`. Write operations are generally fine to use the client directly from handler code.

Reply:
Will do. I haven't forgotten about this, but I've left it for now while I iterate on a few other tasks.

Reply:
I've moved everything over, which also makes testing my changes much easier

Reply:
I think it still makes sense to move this specific call to `pull.Context`, if only for clarity. I'd define it as a `HasReveiwers()` method that is implemented by listing reviewers, making the intent behind the list call more evident.

Reply:
oh, doh. I didn't move this specific call.